### PR TITLE
 Consistent Usage of ES6+ Code

### PR DESCRIPTION
**Pull request by DomAmato**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Description of change

other ES6+ features are used such as `const` so for consistency things like arrow functions, property shorthand, and using includes over indexOf were changed.

### CODE REVIEW

**Comment by cos** [comment]
@leungwensen, is this PR useful?

**Comment by dengfuping** [comment]
- @DomAmato Sorry to close it, because we are working on version `4.x` with `TypeScript` support and have no plan to refactor coding style for version `3.x`.

File: src/geom/shape/schema.js [hunk 1]
```diff
@@ -105,9 +105,7 @@ function _sortValue(value) {
     value = [ value ];
   }
   // 从大到小排序
-  const sorted = value.sort(function(a, b) {
-    return a < b ? 1 : -1;
-  });
+  const sorted = value.sort(function(a, b) { return a < b ? 1 : -1; });
```
Comment (simaQ) [inline src/geom/shape/schema.js]: here is not changed

`value.sort((a, b) => a < b ? 1 : -1);`

I will modify it myself.

**Comment by simaQ** [comment]
+1
Thank you very much for your help. After considering we decided to adopt your PR.

**Pull request merged**